site/en/docs/android/custom-tabs/index.md

```diff
@@ -3,93 +3,117 @@ layout: "layouts/doc-post.njk"
+
+<figure>
+{% Video preload=true, loop=true, playsinline=true, autoplay=true, src="video/DXqUldooyJOUnj3qXSYLHbUgUI93/sIeKPXwrHXdCXtGRrv2Q.mp4", width="350", height="730", class="screenshot" %}
+  <figcaption>A mobile browser, transiting between screens, ending with a web site loaded in a Custom Tab</figcaption>
+</figure>
+
+Modifing the toolbar color to match your apps branding
+
+<figure>
+{% Video loop=true, playsinline=true, autoplay=true, src="video/DXqUldooyJOUnj3qXSYLHbUgUI93/kQ0LUuUdcWFM34IPg5I6.mp4", width="350", height="730", class="screenshot"  %}
+  <figcaption>A mobile browser, transitioning to a Custom Tab with matching colors to the website</figcaption>
+</figure>
+
+And that color consistency can stay with your app, even if they switch between Light and Dark themes
+
+<figure>
+{% Video loop=true, playsinline=true, autoplay=true, src="video/DXqUldooyJOUnj3qXSYLHbUgUI93/YBinAgwhx0kFizQWrrEr.mp4", width="350", height="730", class="screenshot"  %}
+  <figcaption>And that color consistency can stay with your app, even if they switch between Light and Dark themes</figcaption>
+</figure>
+
+You can add custom actions and entries to the browser's toolbar, and menus
+
+<figure>
+{% Video loop=true, playsinline=true, autoplay=true, src="video/DXqUldooyJOUnj3qXSYLHbUgUI93/QFiyUPGANEvjVqfsujF4.mp4", width="350", height="730", class="screenshot"  %}
+  <figcaption>A Custom Tab showing its menu, with custom entries</figcaption>
+</figure>
+
+You can even control the launch height of the Custom Tab, enabling things like
+streaming your videos while interacting with your web store.
+
+<figure>
+{% Video loop=true, playsinline=true, autoplay=true, src="video/DXqUldooyJOUnj3qXSYLHbUgUI93/lsyAKIKYdD87QjSiSQOw.mp4", width="350", height="730", class="screenshot"  %}
+  <figcaption>A partial Custom Tab opening with a set height</figcaption>
+</figure>
+
+That is far from everything - Custom Tabs are very powerful, and under active development. Each browser needs to add support for these features as they become available. While nearly all have some level of support, it is important to know what may or may not be available in your user's browsers. We created a [feature comparison table](/docs/android/custom-tabs/browser-support) to quickly check the availability of the different features across popular Android browsers.
+
+You can test this now with our [sample][1] on GitHub.
+
+## When should I use Custom Tabs?
+
+There is no single "correct" way to load web content. In certain situations,
+WebView is going to be the right thing to use. For example, if you are
+exclusively hosting your own content inside your app, or if you need to inject
+javascript directly from your app. If your app directs people to URLs outside
+domains, the built in shared state in Custom Tabs means they are likely a
+better choice. Other strengths of Custom Tabs include
+
+1. Security: Custom Tabs use Google's Safe Browsing to protect the user and the
```

Comment:
Here's how conventional formatting for this would look. This is non-standard markdown. GitHub isn't rendering it, but our site will using &lt;dl> tags its children.

Security
  : Custom Tabs use Google's ...
Performance optimization
  : The browser:
    * Pre-warms in the background....
    * Speeds up page load times...

In any case, ordered lists should only be used for procedures.

Reply:
This is also a place where parallel text should have parallel grammar. 

Would you like me to take a stab at arranging this?

Reply:
apologizes, I didn't write a majority of this doc, I mostly was updating screenshots and cleaning up some language.

I had attempted to migrate this to dl/dt but given the lack of clear titles (and therefore any relevant list member) on the later half I was at a loss to the formatting

```md
Security
: Custom Tabs use Google's Safe Browsing to protect the user and the device from dangerous sites.

Performance optimization
: Pre-warming of the Browser in the background, while avoiding stealing resources from the application.
: Speed up the page load time by speculatively loading URLs in advance.

Lifecycle management
: Apps launching a Custom Tab won't be evicted by the system during the Tabs use - its importance is raised to the "foreground" level.

Shared cookie jar and permissions model so users don't have to sign-in to sites they are already connected to, or re-grant permissions they have already granted.

Browser features like Data Saver are shared, if enabled - loading content faster and cheaper.

Synchronized AutoComplete across devices for better form completion.

Users can easily return to app with an integrated back button.
```

ends up looking like [this]https://user-images.githubusercontent.com/465414/236572079-4082a826-ee23-4ce7-8056-14613b03bd99.png). 


if the language needs updating, I would love to see your stabs

Reply:
This is how our site renders it. https://developer.chrome.com/docs/extensions/mv3/service_workers/events/#events

I second thought, I'd like to call this "would be nice to fix, but shouldn't block."

Reply:
Sorry, my comment meant that I have the titles of the sections (ie the equivalent of chrome.* in the sections you mentioned) but not the details element below it. because of that, the markdown we use won't recognize the sections as individual titles.  I would love for the formatting to be consistent, but am not sure what the recommended way to do that would be.